controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/configserver/ConfigServerApiHandler.java

```diff
@@ -102,7 +103,7 @@ private HttpResponse root(HttpRequest request) {
         ZoneList zoneList = zoneRegistry.zones().reachable();
 
         Cursor zones = root.setArray("zones");
-        Stream.concat(Stream.of(CONTROLLER_ZONE), zoneRegistry.zones().reachable().ids().stream())
+        Stream.concat(Stream.of(controllerZone), zoneRegistry.zones().reachable().ids().stream())
```

Comment:
We should consider adding a method to list zones including the controller zone so we avoid having to do this concat everywhere

Reply:
Yes, I thought about, but it wasn't immediately obvious how to best add this to `ZoneList`. Will do it in a future PR.